AirCasting/Extensions/Array+byRemovingDuplicates.swift

```diff
@@ -0,0 +1,50 @@
+// Created by Lunar on 30/08/2022.
+//
+import Foundation
+
+extension Numeric {
+    func decrement() -> Self {
+        self - 1
```

Comment:
In my opinion incrementing makes a bit more sense logically. If we have a sorted array with duplicated values like: `[1, 2, 2, 3]`, then for the second `2` the appendBuffer function would change it to `0`, and the caller of this function would have to sort it to have the order right. With incrementing, the order of elements would stay the same. But it is more of a neat pick. If you prefer to decrement, I think it works okay as well.